src/components/next/demo/nested/index.tsx

```diff
@@ -0,0 +1,53 @@
+import { Component, h, Host, VNode } from "@stencil/core"
+import { data } from "./data"
+
+@Component({
+	tag: "smoothly-next-demo-nested",
+	styleUrl: "style.css",
+	scoped: true,
+})
+export class SmoothlyNextDemoNested {
+	render(): VNode | VNode[] {
+		return (
+			<Host>
+				<smoothly-next-table columns={8}>
+					<smoothly-next-table-head>
+						<smoothly-next-table-row>
+							<smoothly-next-table-cell>Id</smoothly-next-table-cell>
+							<smoothly-next-table-cell>Registered</smoothly-next-table-cell>
+							<smoothly-next-table-cell>Name</smoothly-next-table-cell>
+							<smoothly-next-table-cell>Age</smoothly-next-table-cell>
+							<smoothly-next-table-cell>Balance</smoothly-next-table-cell>
+							<smoothly-next-table-cell>EyeColor</smoothly-next-table-cell>
+							<smoothly-next-table-cell>Gender</smoothly-next-table-cell>
+							<smoothly-next-table-cell>Company</smoothly-next-table-cell>
+						</smoothly-next-table-row>
+					</smoothly-next-table-head>
+					<smoothly-next-table-body>
+						{data.map(entry => (
+							<smoothly-next-table-expandable-row>
+								<div slot={"detail"}>
```

Comment:
in most cases these divs should be `<smoothly-lazy>` the attribute `content` specified for even better performance.